aten/src/ATen/native/mkldnn/xpu/Blas.cpp

```diff
@@ -0,0 +1,422 @@
+#include <ATen/WrapDimUtilsMulti.h>
+#include <ATen/native/Resize.h>
+#include <ATen/native/mkldnn/xpu/BlasImpl.h>
+#include <torch/library.h>
+
+namespace at {
+namespace native {
+namespace xpu {
```

Comment:
Since the code is under `mkldnn`, may I know why the namespace does not contain `onednn`?

Reply:
I choose this is because `at::native::xpu` is aligned with `at::native::cuda` and `at::native::cpu` namespace, which is classified through device backend.
`at::native::onednn` would be a specific namespace for onednn related operators, which is aligned to `at::native::cudnn`